scripts/pi-hole/js/settings.js

```diff
@@ -169,7 +169,7 @@ $(".api-token").confirm({
   text: "Make sure that nobody else can scan this code around you. They will have full access to the API without having to know the password. Note that the generation of the QR code will take some time.",
   title: "Confirmation required",
   confirm: function () {
-    window.open("scripts/pi-hole/php/api_token.php");
+    window.open("scripts/pi-hole/php/api_token.php", "_blank");
```

Comment:
Just thinking out loud: Maybe this could be replaced by an iFrame embedded in a modal dialog? Like it was done here:
https://github.com/pi-hole/AdminLTE/pull/1231

Reply:
Oh yeah that is a better idea, the experience felt off when I was testing it.

Reply:
Wanted to update I'm still on this, will probably get to it tonight/tomorrow.

Reply:
No need to rush, take your time.